web/src/pages/barcoding.js

```diff
@@ -1,9 +1,335 @@
-import React from 'react'
-
-export default function BarcodingPage() {
-    return (
-        <div>
-            <h1>this is the Barcoding Page</h1>
-        </div>
-    )
+import React from 'react';
+import tw from 'twin.macro';
+import Img from 'gatsby-image';
+import { graphql } from 'gatsby';
+import styled from 'styled-components';
+
+import Layout from '../components/Layout'
+
+const HeroStyle = styled.div`
+  width: 100vw;
+  ${tw`w-screen`};
+  top: -160px;
+  margin-bottom: 200px;
+
+  .gatsby-image-wrapper {
+    top: 100px
+
+  }
+  @media screen and (max-width: 480px) {
+   .gatsby-image-wrapper {
+     width: 100vw;
+     top: 150px
+   }
+   margin-bottom: 200px;
+ }
+//  @media screen and (max-width: 700px) and (min-width: 400px) {
+//   .gatsby-image-wrapper {
+//     width: 100vw;
+//   }
+//   ${tw`w-screen`};
+// }
+`;
+
+const ImgFlex = styled.div`
+  ${tw`grid grid-cols-2 gap-8 mx-24 mb-24`};
+
+  @media screen and (max-width: 420px) {
+   .mask, .welcome {
+     width: 165px;
+   }
+   ${tw`mx-6 mb-6`}
```

Comment:
Can you add these to line 35? Tailwind let's you prepend any of their classes with their build in breakpoints like `md:mx-10`.

I think what you'll want to do is have something like:
`mx-6 md:mx-0 mb-6 md:mb-0` (not sure if `md:` is the right breakpoint, you'll need to check)

I know it might seem like a lot to add a style just to remove it at the next screen size but I think it communicates clearly what the intention is (as opposed to seeing different classes added in unique media queries.

Let me know if you have any questions.